hammerdb/ch_benchmark.py

```diff
@@ -0,0 +1,490 @@
+#!/usr/bin/python
+
+import sys
+import os
+import random
+import atexit
+import signal
+import subprocess
+
+from os.path import expanduser
+from threading import Thread
+from time import sleep
+import time 
+
+ch_queries = [
```

Comment:
So, there is not any external parameters that ch queries get?

Reply:
I dont think there is